portfolio/src/main/webapp/script.js

```diff
@@ -78,3 +77,66 @@ function deleteComment(comment) {
   params.append('id', comment.id);
   fetch('/delete-comment', {method: 'POST', body: params});
 }
+
+function loadMap() {
+  var sanFrancisco = {lat: 37.7749, lng: -122.4194};
+  const cafeMap = new google.maps.Map(
+    document.getElementById("map"),
+    {center: sanFrancisco, zoom: 12});
+
+  const philzMarker = new google.maps.Marker({
+    position: {lat: 37.793949, lng: -122.398062},
+    map: cafeMap, 
+    title: "Philz Coffee"
+  });
+
+  const philzInfoWindow = new google.maps.InfoWindow({
+    content: "I like the Iced Coffee Rose!"
+  });
```

Comment:
you seem to be doing the same thing for each location. what you can do to make it more readable is make this into a helper function that takes in the lat, lng, map, title, content. and repeatedly call it for each location.